src/System.Linq.Expressions/tests/BinaryOperators/Arithmetic/BinaryAddTests.cs

```diff
@@ -282,6 +336,54 @@ private static void VerifyShortAdd(short a, short b)
             }
         }
 
+        private static void VerifyShortAddOvf(short a, short b)
```

Comment:
The ET part might work with generic a, b and typeof(T), but I am not sure the C# part of validation like "csResult = checked((short)(a + b));"   or   "" would work the same way with generics. 
I guess, if I thought about this before, it could save me some time typing, but now, considering that these are tests and are unlikely to change, I think duplicate code is acceptable here. 
If I have to change these significantly, I will consider a generic version.